inst/examples/dashboard/header.R

```diff
@@ -53,11 +53,92 @@ farItems <- list(
   )
 )
 
-command_bar <- CommandBar(items = items,
-  farItems = farItems,
-  style = list(width = "100%"))
+appsilon_breakpoints <- breakpointSystem(
+  "appsilon-breakpoints",
+  breakpoint("xs", min = 320),
+  breakpoint("s", min = 428),
+  breakpoint("m", min = 728),
+  breakpoint("l", min = 1024),
+  breakpoint("xl", min = 1200)
+)
+
+header <- gridPanel(
+  id = "app_header",
+  class = "mobile-collapsed",
+  breakpoint_system = appsilon_breakpoints,
+  areas = list(
+    default = c(
+      "logo . info mobile_controls",
+      "separator separator separator separator",
+      "title title title title",
+      "menu menu menu menu",
+      "cta cta cta cta"
+    ),
+    l = "logo separator title mobile_controls . menu info cta"
+  ),
+  columns = list(
+    default = "auto 1fr auto auto",
+    l = "auto 1px auto auto 1fr auto auto auto"
+  ),
+
+  rows = list(
+    default = "auto auto auto auto auto",
+    l = "40px"
+  ),
+
+  gap = list(
+    default = "0px",
+    l = "16px"
+  ),
 
-logo <- img(src = "appsilon-logo.png", class = "logo")
-title <- div(Text(variant = "xLarge", "shiny.fluent"), class = "title")
+  logo = img(src = "appsilon-logo.png", class = "logo"),
 
-header <- tagList(logo, title, command_bar)
+  separator = div(class = "app_header_vertical_separator mobile-toggled"),
+
+  title = div(Text(variant = "xLarge", "shiny.fluent"),
+              class = "title app_header_title mobile-toggled"
+  ),
+
+  command_bar <- CommandBar(items = items,
+                            farItems = farItems,
+                            style = list(width = "100%")),
```

Comment:
Please use the following indentation style; ditto for the rest of the file.
```suggestion
  command_bar <- CommandBar(
    items = items,
    farItems = farItems,
    style = list(width = "100%")
  ),
```